### PR TITLE
Updated shadow.md, fixed Role handbook url

### PR DESCRIPTION
**Pull request by ParvJain**
Fixed Role Handbook url to point to right directory

/kind cleanup

### CODE REVIEW

File: release-team/shadows.md
Comment (hasheddan) [inline release-team/shadows.md:13]: I think this link may be intentional since it lands you on the README for a summary of release team roles

Comment (ParvJain) [reply release-team/shadows.md:13]: I might be wrong here, but IMO either anchor or link is misleading, do you think it would be good idea to change anchor from "Role Handbooks" to something else as an alternative?

Comment (hasheddan) [reply release-team/shadows.md:13]: @ParvJain sorry for the delay here! Looking back here, I think your update is appropriate :)

**Review by hasheddan — APPROVED** [review]
/lgtm

File: release-team/shadows.md
Comment (justaugustus) [inline release-team/shadows.md:13]:
```suggestion
Please read the [Role Handbooks](/release-team/role-handbooks) for those roles before filling out the form.  Some handbooks will have a specific Shadow section that addresses what background Shadows are expected to have and what tasks they do.
```

Use absolute paths when linking within the repo, but outside of the directory.
ref: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md#links

**Review by justaugustus — CHANGES_REQUESTED** [review]
@ParvJain -- one nit to address

**Comment by jeremyrickard** [comment]
@ParvJain hey there 👋  do you have time to make the updates requested up above?

**Comment by palnabarun** [comment]
@ParvJain -- one last thing, can you please squash the commits? :)

/hold for squash

**Comment by soniasingla** [comment]
@ParvJain hi, do you have time to make the changes requested above? :)

**Comment by palnabarun** [comment]
Letting tide to do the squash magic. Added the `tide/merge-method-squash` label manually.

/hold cancel

**Comment by palnabarun** [comment]
/lgtm

Thank you @ParvJain for your changes. 👍🏽

**Pull request merged**
